src/components/Select/Options.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+    import type { Option } from './types.ts';
+
+    export let open: boolean;
+    export let options: Option[];
+    export let value: Option;
+    let dialog: HTMLDialogElement;
+
+    $: if (dialog && open) dialog.showModal();
+
+    function select(option: Option) {
+        value = option;
+    }
+    function close() {
+        open = false;
+    }
+    function outsideClick() {
+        dialog.close();
+    }
```

Comment:
It's a little cramped here.
```suggestion
    function select(option: Option) {
        value = option;
    }

    function close() {
        open = false;
    }

    function outsideClick() {
        dialog.close();
    }
```